pkg/mesh/topology.go

```diff
@@ -19,6 +19,8 @@ import (
 	"net"
 	"sort"
 
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
```

Comment:
By convention we normally separate third party inputs from standard library imports and imports from this module.
```suggestion
	"github.com/go-kit/kit/log/level"

```